go/fixchain/fix_error.go

```diff
@@ -39,6 +44,8 @@ func (e FixError) TypeString() string {
 		return "CannotFetchURL"
 	case FixFailed:
 		return "FixFailed"
+	case PostFailed:
```

Comment:
PostFailed is used when the call to http.Client.Post() returns an error.  LogPostFailed is used when the response from the log contains a status code other than 200.